AssetTracking/AssetTracking/appsettings.json

```diff
@@ -1,8 +1,34 @@
 {
+
+  "AzureAd": {
+    "AllowedHosts": "*",
+
+    "CallbackPath": "/signin-oidc",
+
+    "BaseUrl": "https://localhost:44371",
+
+    "ClientId": "c2d47c42-f92d-49d1-9390-7fe44779f665",
+
+    "ClientSecret": "Ye[1Rpy.9Jv**u8rfUIprVLSxxKQH6Ct", // This sample uses a password (secret) to authenticate. Production apps should use a certificate.
```

Comment:
Use ASP.NET Core client secret manager to manage app. confidential info. (`ClientSecret` and `ClientId`)
[https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-2.2&tabs=windows#secret-manager
](url)

Reply:
The link doesn't work

Reply:
Okay will read more into it.